WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.kt

```diff
@@ -154,7 +154,11 @@ class MySitesPage {
 
     fun goToStats(): StatsPage {
         goToMenuTab()
-        clickQuickActionOrSiteMenuItem(R.id.quick_action_stats_button, R.string.stats)
+        val statsButton = Espresso.onView(Matchers.allOf(
+            ViewMatchers.withText(R.string.stats),
+            ViewMatchers.withId(R.id.my_site_item_primary_text)
+        ))
+        WPSupportUtils.clickOn(statsButton)
```

Comment:
Instead of trying to tap the `Stats` button using one of the two alternative locators (one for `Stats` button in `Quick Links`, and another one in the classical` Menu`) I went for using one in classical menu, which is always there.